src/client/components/board/board.css

```diff
@@ -7,4 +7,15 @@
 .status-bar {
   border-bottom: 1px solid #ccc;
   padding: 5px;
+}
+
+.license {
+  text-align: center;
+  margin-right: 20px;
```

Comment:
I believe it is sufficient to have only a singe line at the bottom right. So, I would remove the margins and the width.